src/lib/api/agent.ts

```diff
@@ -36,6 +36,32 @@ export async function add(dept_id: Dept['dept_id'], uid: User['user_id'], head:
     }
 }
 
+/** Removes an {@linkcode Agent} from the {@linkcode Dept}. Returns `true` if successful, `false` otherwise. */
+export async function remove(dept_id: Agent['dept_id'], uid: Agent['user_id']) {
```

Comment:
Using `snake_case` in JavaScript itself is a little strange.
```suggestion
export async function remove(did: Agent['dept_id'], uid: Agent['user_id']) {
```

Reply:
Woops, my bad. Thanks!